src/estimagic/optimization/tranquilo/tranquilo.py

```diff
@@ -158,14 +157,14 @@ def _tranquilo(
 
     bounds = Bounds(lower=lower_bounds, upper=upper_bounds)
 
-    if trustregion_shape is None:
-        trustregion_shape = "sphere" if not bounds.has_any else "cube"
-
     if sampler is None:
-        sampler = f"optimal_{trustregion_shape}"
+        sampler = "optimal_sphere"
```

Comment:
In the long run this should just be `"optimal"`. Will you do that in the next PR?